Python/pywarpx/picmi.py

```diff
@@ -1092,7 +1092,24 @@ def initialize_inputs(self):
 
 
 class LabFrameFieldDiagnostic(picmistandard.PICMI_LabFrameFieldDiagnostic):
+    def init(self, kw):
+        self.use_new_BTD = kw.pop('warpx_new_BTD', False)
```

Comment:
Should we add a docstring `"""..."""` to describe the kwargs that are possible to be passed here?

Reply:
Good idea, though it has to be a bit kludgy to retain the full doc string. I added it for this case. If this looks good, a later PR can add the doc for the rest of the classes.